mypy/checkexpr.py

```diff
@@ -1,7 +1,7 @@
 """Expression type checker. This file is conceptually part of TypeChecker."""
 
 from mypy.ordered_dict import OrderedDict
-from contextlib import contextmanager
+from contextlib import contextmanager, nullcontext
```

Comment:
CI will tell you that 3.6 doesn't have nullcontext. Easy enough to redefine though

Reply:
🙏